stuff/bootstrap/panels.less

```diff
@@ -214,7 +214,7 @@
 }
 
 
-// Collapsable panels (aka, accordion)
```

Comment:
I'd like to keep changes in bootstrap .less files to the minimum, so diffing and porting changes is easier.

Reply:
I tried to filter out dependencies but I missed this one.